src/dosbox.cpp

```diff
@@ -916,17 +916,26 @@ const char *filter_on_or_off[] = {"on", "off", 0};
 	                  "  on:   Filter the output (default).\n"
 	                  "  off:  Don't filter the output.");
 
-	// Disney Audio emulation
-	secprop->AddInitFunction(&DISNEY_Init,true);//done
-
-	Pbool = secprop->Add_bool("disney", when_idle, true);
-	Pbool->Set_help("Enable Disney Sound Source emulation. (Covox Voice Master and Speech Thing compatible).");
-
-	Pstring = secprop->Add_string("disney_filter", when_idle, "on");
-	Pstring->Set_help("Filter for the Disney audio output:\n"
+	// LPT DAC device emulation
+	secprop->AddInitFunction(&LPT_DAC_Init, true);
+	const char *lpt_dac_types[] = {"none", "disney", "covox", "ston1", "off", 0};
+	pstring = secprop->Add_string("lpt_dac", when_idle, lpt_dac_types[0]);
+	pstring->Set_help("Type of DAC(s) to plug into the parallel port:\n"
+	                  "  disney:   Disney Sound Source, in mono at 7 kHz.\n"
+	                  "  covox:    Covox Speech Thing, in mono at the mixer's rate.\n"
+	                  "  ston1:    Stereo-on-1 DAC, in stereo up to 30 kHz.\n"
```

Comment:
Not a fan of this, can we just call it `covox-stereo`? Stereo-on-1 DAC is just a name some Vogons user came up with, I don't think it's "definitive", and it's a bit confusing... "Stereo Covox" is very easy to understand, and I don't think explicitly stating that "stereo is implemented in a single interface" adds a lot of value... We don't say for OPL2 that it's "stereo OPL on a single Sound Blaster card" or something. People just say Dual OPL, or Stereo OPL.

Reply:
Actually, I read this again, and now I'm confused...
```
Covox Speech Thing, in mono at the mixer's rate.
```

The sample rate for the Covox can be freely chosen. Does the new implementation still support this? I know for certain that the old one did; I tested Pinball Dreams with various quality settings to tweak the filter, which set different Covox sample rates.

So 1) we definitely should support variable Covox rates set by software, 2) I'd remove _all_ references to sample rates from these descriptions. It just muddies the water, and we don't mention them for other sound cards at all, so why do it here? And it would get too complex to do it properly (e.g. how about the variable GUS sample rates, then there's always the difference between actual mixer sample rate, and the "virtual" sample rate, achieved by the ZoH upsampler, etc.) If we want to go into details about sample rates, I think the documentation is the place for that.

Reply:
Absolutely- agree on all points, and will drop references to the frequency and mono/stereo wording. The covox speech thing only was a mono device, so I'm only adding confusion by mentioning that point (some users might wonder if there's a stereo version or if the emulation is just lacking). 


Yes, all three DAC models can handle any sampling rate fed into them without time stretching or compression, although quality wise, were still capped out by the mixer rate (some users reportedly had fast enough machines to feed their covox a 60 kHz signal, but in our case we're running Cox's channel at 48 kHz (sending it 60 kHz would still sound fine, but I suppose we're losing a bit of those ultrasonic frequencies)

Reply:
> some users reportedly had fast enough machines to feed their covox a 60 kHz signal, but in our case we're running Cox's channel at 48 kHz (sending it 60 kHz would still sound fine, but I suppose we're losing a bit of those ultrasonic frequencies)

Yeah, and it doesn't matter that we downsample from 60k to 44.1/48k because no one will notice. Most humans lose the ability to hear anything above ~18k at around 18-20 years of age. Then it gets down to about 14-15k around middle age (40+), and as low as 10-12k by old age (60+).

I for one cannot hear a thing (as in zero, absolutely nothing) above 14k at 43, and my wife's hearing caps out around 12k, we've measured it.

So those greater than 20k sampling rates exist mostly for the "audiophile feel-good" factor... but in most cases it's a psychological feel-good factor of "knowing" things are working at a higher rate, because they clearly wouldn't be able to physically register the sensations (well, theoretically maybe there are some weird genetic mutations among humans who can hear above 20k, even later in life... but I don't think all those audiophiles claiming to be able to "hear" outrageously high frequencies belong to that group... they belong to another "special" group, if you ask me 😝)

Yes, I'm an "objectivist", according to audiophile nomenclature 🤣

Reply:
Went with order 2 @ 9 KHz; sounds decent w/ the sampling of .MOD files I threw at it, without chopping off too much of the cymbals and hihat samples.